contrib/graph-places.py

```diff
@@ -0,0 +1,154 @@
+import argparse
```

Comment:
Since this is meant to be run as a script, we should add a shebang and set the executable bit.